### ISSUE TITLE
Icon button group got wrong border-color

### ISSUE THREAD

**Issue by Jupi007** [sulu/sulu#6883]
| Q | A
| --- | ---
| Sulu Version | Tested on 2.5, but it should affect also 2.4

# Actual Behavior

When a disabled icon button is located against a disabled one, the middle border use the wrong color:

![Capture d’écran du 2022-11-01 21-06-36](https://user-images.githubusercontent.com/36476595/199574296-40a49ec0-43a7-49b8-b37a-daab73c2a531.png)

# Expected Behavior

The middle border should the normal color:

![Capture d’écran du 2022-11-01 21-06-41](https://user-images.githubusercontent.com/36476595/199574289-0e778c9a-f0e4-4f33-9d82-94f1dcec00fd.png)

# Steps to Reproduce

Go to any admin page with pagination and at least 2 pages.

# Possible Solutions

```scss
// src/Sulu/Bundle/AdminBundle/Resources/js/components/Button/button.scss

.icon {
  // ...
  &:not(:disabled) + &:disabled {
    border-left-color: $iconBorderColor;
  }
  // ...
}
```

This fix the problem, I tested it with various combination of a group of 3 buttons.
But I have no idea if it can have side effect, as I don't know well how this component is used in the sulu admin.

**Comment by alexander-schranz** [comment]
@Jupi007 thx for report.

I think we should handle this in the ButtonGroup component and not in the `icon` component: https://github.com/sulu/sulu/blob/2.5/src/Sulu/Bundle/AdminBundle/Resources/js/components/ButtonGroup/buttonGroup.scss

Is there something like your fix also possible?

**Comment by Jupi007** [comment]
I think you're right, it is the cleaner solution, and it shouldn't cause any side effect.
It is just less simple to fix:

```scss
// src/Sulu/Bundle/AdminBundle/Resources/js/components/ButtonGroup/buttonGroup.scss

.button {
  // ...    
    &:not(:first-child) {
        border-left-width: 0;
    }

    &:disabled:not(:last-child) {
        border-right-width: 0;
    }

    &:disabled + & {
        border-left-width: 1px;
    }
}
```

**Before:**

![image](https://user-images.githubusercontent.com/36476595/199591156-2a29c4c2-43c5-41bb-abe4-2f7877990aed.png)

**After:**

![Capture d’écran du 2022-11-02 20-55-58](https://user-images.githubusercontent.com/36476595/199590868-8d67315f-4182-4fce-9f26-9268e3c7fcd1.png)

**Comment by alexander-schranz** [comment]
@Jupi007 sounds good this way will you open a PR?

**Comment by Jupi007** [comment]
Ofc, I will do one 👍

**Comment by alexander-schranz** [comment]
fixed in #6885 thank you!
